clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/ManifestProvider.java

```diff
@@ -27,6 +27,17 @@
 
   T getManifest(String account, String location, String name);
 
+  T getManifest(String account, String location, String name, boolean includeEvents);
```

Comment:
Given that this is only called in 2 places, it might be simpler to just always require the boolean flag.  The only remaining location that would call the old version is `KubernetesV2JobProvider` which also doesn't need events so should just pass in `false`.